ros2_batch_job/__main__.py

```diff
@@ -266,16 +266,36 @@ def __call__(self, parser, namespace, values, option_string=None):
             setattr(args, name, space_directory)
     return args
 
+def get_lcov_version():
+    try:
+        result = subprocess.run(['lcov', '--version'],
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE,
+                                text=True, check=True)
+        version_output = result.stdout.strip().replace('lcov: LCOV version ', '')
+        return version_output
+    except subprocess.CalledProcessError as e:
+        return ""
 
 def process_coverage(args, job):
     print('# BEGIN SUBSECTION: coverage analysis')
     # Capture all gdca/gcno files (all them inside buildspace)
     coverage_file = os.path.join(args.buildspace, 'coverage.info')
+
+    version = get_lcov_version()
+    lcov_arguments = []
+    if version.startswith('2'):
+        lcov_arguments = ['--ignore-errors', 'inconsistent,inconsistent',
+                          '--ignore-errors', 'mismatch,mismatch',
+                          '--ignore-errors', 'negative,negative',
+                          '--ignore-errors', 'unused,unused',
+                          '--ignore-errors', 'empty,empty']
```

Comment:
I'm not against this, but can you explain more why we need to ignore all of these errors?  I think it would be best to have a comment here with that, so future readers know why.

Reply:
I was just launching the CI and fixing the errors.